src/api/functions.ts

```diff
@@ -135,7 +124,7 @@ async function createFunctionVersion(
   client: GotClient
 ): Promise<VersionResource> {
   if (fn.access === 'private') {
-    throw new Error(`Function ${fn.functionPath} cannnot be "private". 
+    throw new Error(`Function ${fn.name} cannnot be "private".
 Please rename the file "${fn.name}" to "${fn.name.replace(
```

Comment:
Hmm... I missed this one. :) 

If you agree with the overall change I'll change the error message to something like

`Please make function public/protected by renaming the file (if you're using @twilio/serverless-api) or changing the configuration of it`